app/grandchallenge/algorithms/models.py

```diff
@@ -69,6 +69,38 @@
 JINJA_ENGINE = sandbox.ImmutableSandboxedEnvironment()
 
 
+class AlgorithmInterfaceManager(models.Manager):
```

Comment:
Should the create method also be overridden?

Reply:
Good idea. then we can handle setting inputs and outputs in the custom create method, and don't need to repeat the same 3 lines everywhere we create a new interface. I can't use that in the migration, but that's ok.